scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -151,41 +161,36 @@ else
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
-    sleep 15 # maybe the mailbox controller is slow, give it a chance
+	echo "maybe the mailbox controller is slow, give it a chance"
+    sleep 15
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
-    sleep 15 # maybe the mailbox controller is slow, give it another chance
+	echo "maybe the mailbox controller is slow, give it another chance"
+    sleep 15
 fi
 
 if ! kubectl "${kubectl_flags[@]}" get Workspace "$mbwsname" &> /dev/null; then
     echo "$0: did not find the mailbox workspace $mbwsname; is the mailbox controller running?" >&2
     exit 5
 fi
 
-# Now that workspace _exists_, but is it _ready_?
+echo "now that workspace _exists_, but is it _ready_?"
 sleep 5
 
 if ! kubectl ws "${kubectl_flags[@]}" "$mbwsname" &> /dev/null; then
+	kubectl ws .
```

Comment:
This is debugging to remove, right?

Reply:
These waits are inherited from previous code which waits for the mailbox-controller. In the previous lines 154 and 158, comments say that 'maybe the mailbox controller is slow'. Do we need to keep them?

Reply:
My comment is about the `kubectl ws .`

Reply:
I see. Addressed in #1338.